src/index.ts

```diff
@@ -94,12 +105,12 @@ export async function installLatest(ui: UI) {
     const release = await Github.latestRelease();
     const asset = await Github.chooseAsset(release);
     ui.clangdPath = await Install.install(release, asset, abort, ui);
-    ui.promptReload(`clangd ${release.name} is now installed.`);
+    ui.promptReload(ui.localize(`clangd {0} is now installed.`, release.name));
```

Comment:
nit, here and elsewhere: since we're no longer using `${...}` inside these strings, the delimiters can be changed from `` ` `` (backticks) to `'` (regular single quotes)